react-native/android/app/proguard-rules.pro

```diff
@@ -15,3 +15,46 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
```

Comment:
yeah this looks fine, we weren't using progaurd yet anyways. It's for optimizing the size of the app and removing dead code.